aicsimageio/readers/tiff_glob_reader.py

```diff
@@ -244,6 +243,9 @@ def indexer(x: str) -> pd.Series:
             with self._fs.open(self._path) as open_resource:
                 with TiffFile(open_resource) as tiff:
                     if tiff.is_shaped:
+                        print(f"Description: {tiff.is_shaped}")
+                        print(dir(tiff))
+                        print(tiff.shaped_metadata)
```

Comment:
Should this be removed?

Reply:
Will remove as soon as I commit the new tifffile version: https://github.com/cgohlke/tifffile/issues/127#issuecomment-1109402447